qip-0011.md

```diff
@@ -0,0 +1,209 @@
+be chosen to make the initial value of Qi to be near parity with the dollar, Euro, or Pound.
+Also, note here that $k_{qi}$ is a function of $\epsilon$ which is the relative projected change
+in efficiency of FET based GPUs. This is done to prevent changes in efficiency of GPUs from causing
+significant short term deflation in the purchasing power of Qi.  
+
+### Quai Reward ###
+
+The reward fuction for Quai will take the form of:
+
+$$ Quai(k_{quai}(\bar{R}\_{hash}),diff) = 2^{-k_{quai}(\bar{R}_{hash})} * log_2(diff) $$
+
+The astutue reader will notice that a simple model has been introduced, namely raising $k_{quai}$
+to the power of 2. This will allow a simple controller to maintain stability
+with fixed controller gains over orders of magnitude changes in difficulty.
+
+### Projected  Efficiency ###
+
+The projected GPU efficiency is introduced to help Qi be more strictly reflective of the electricity
+cost component in mining.  This will help Qi maintain a more stable purchasing power compared
+to having a reward function that does not adjust with GPU efficiency. Again the primary goal
+being to have a low enough volotility in the change in purchasing power to allow goods and
+services to be easily denominatable in Qi. Thus, Qi could be used as a unit of account.
+
+The projected GPU efficiency curve should be fitted to relect the median prediction from:
+
+Marius Hobbhahn and Tamay Besiroglu (2022), ["Predicting GPU Performance"](https://epochai.org/blog/predicting-gpu-performance)
+
+The curve proposed is as follows:
+
+![projected-gpu-performance](qip-0011/predicting-gpu-performance.png "Projected GPU performance")
+> Image 1: The projected performance of FET based GPUs
+
+Extracted Data 
+```
+X | Y
+2002.7,10.75
+2014.1,12.04
+2024.75,13.36
+2028.9,13.87
+2030.8,14.10
+2032.6,14.23
+2036.7,14.31
+2051.4,14.31
+```
+The fit derived from the expected performance data is as follows:
+
+$$
+f(x) = \begin{cases} 
+-229.16 + 0.119779 x & \text{for } x < 2032 \\
+14.257 & \text{for } x \geq 2032 \\
+\end{cases}
+$$
+
+Based on the launch date of quai this function will need to be adjusted to account
+for both the start date as well as the conversion between block number and year.
+
+## Controller Design
+
+Both the exchange rate and block reward controller will take the form of a simple
+proportional integral controller.
+
+### Exchange Rate Controller ###
+
+The exchange rate controller input will be the simple moving average of the 
+aggregate hash equivalent demand, $\bar{R}\_{hash}$ for Qi, $D_{qi}$, and Quai, $D_{quai}$ over a period of 100 prime blocks. The setpoint
+of the controller will be to maintain equivalent aggregate demand for Qi and Quai. The hash equivalent value
+needs to be updated with each step. The $\bar{R}_{hash}$ will be computed as follows:
+
+$$ \bar{R}\_{hash} = \sum_{l=n-m}^{n} \frac{D_{quai,l} - D_{qi,l}}{D_{quai,l} + D_{qi,l}} $$
```

Comment:
Why use an inefficient FIR design (lots of memory and subsequent computations) instead of an IIR design (no memory and only a single iterative computation, just like the DAA)?

Reply:
Its not stable. Notice that the values are being recomputed to the relative hash at the current rate rather than taking a lagged average.

Reply:
Which values are getting recomputed? Are you saying $D_{quai,i}$ and $D_{qi,i}$ for a given _i_ will differ over time? I would expect that the demand at block _i_ is the same forever after block _i_ has been accepted.

Reply:
even your PI controller below is implemented as an IIR. Implementation would be a lot simpler for the final form of the controller to be a cascade of two first order IIR filters